website/addons/zotero/model.py

```diff
@@ -1,56 +1,320 @@
-"""
+# -*- coding: utf-8 -*-
+from framework.exceptions import PermissionsError
 
-"""
+from website.addons.base import AddonUserSettingsBase
 
-from bs4 import BeautifulSoup
-import requests
-
-from framework import fields
+from modularodm import fields
 
 from website.addons.base import AddonNodeSettingsBase
+from website.oauth.models import ExternalProvider
+from pyzotero import zotero
+
+from website.addons.citations.utils import serialize_account, serialize_folder
+
+from . import settings
+
+class AddonZoteroUserSettings(AddonUserSettingsBase):
```

Comment:
Same thing here: these models do not need to be prefixed with "Addon". @samchrisinger 

Will change these when I merge.